tests/unit/test_schemas.py

```diff
@@ -0,0 +1,49 @@
+import os
+import subprocess
+
+import pytest
+from ruamel.yaml import YAML
+
+import tmt
+
+PATH = os.path.dirname(os.path.realpath(__file__))
+SCHEMADIR = os.path.join(PATH, "../../tmt/schemas")
+ROOTDIR = os.path.join(PATH, "../..")
+
+# make sure tmt tree is initialized, required when tests run during rpmbuild
+tmt.base.Tree.init(ROOTDIR, 'empty', False)
+
+
+@pytest.fixture
+def schema_and_store():
+    # TODO: tmt package shall provide a helper function
+    # for "load schemas for test/plan/story"
+    schema_file = os.path.join(SCHEMADIR, 'tests.yaml')
+    schema = YAML(typ="safe").load(open(schema_file, encoding="utf-8"))
+
+    schema_store = {}
+
+    for schema_name in ('common', 'core'):
+        schema_file = os.path.join(SCHEMADIR, f'{schema_name}.yaml')
+        store_schema = YAML(
+            typ="safe").load(
+            open(
+                schema_file,
+                encoding="utf-8"))
+        schema_store[store_schema['$id']] = store_schema
+
+    return schema, schema_store
+
+
+@pytest.fixture(params=tmt.Tree('.').tests())
```

Comment:
```suggestion
@pytest.fixture(params=tmt.Tree(ROOTDIR).tests())
```
@psss just spotted this